modules/telemetry/v1/data_block_enums.py

```diff
@@ -0,0 +1,101 @@
+from enum import IntEnum
+
+
+class DataBlockSubtype(IntEnum):
+    """Lists the subtypes of data blocks that can be sent in Version 1 of the packet encoding format."""
+
+    DEBUG_MESSAGE = 0x00
+    ALTITUDE = 0x01
+    TEMPERATURE = 0x02
+    PRESSURE = 0x03
+    ACCELERATION = 0x04
+    ANGULAR_VELOCITY = 0x05
+    GNSS_LOCATION = 0x06
+    GNSS_METADATA = 0x07
+    HUMIDITY = 0x08
+    STATUS = 0x09
+    RESERVED = 0xFF
+
+    def __str__(self):
+        match self:
+            case DataBlockSubtype.DEBUG_MESSAGE:
+                return "DEBUG MESSAGE"
+            case DataBlockSubtype.ALTITUDE:
+                return "ALTITUDE"
+            case DataBlockSubtype.TEMPERATURE:
+                return "TEMPERATURE"
+            case DataBlockSubtype.PRESSURE:
+                return "PRESSURE"
+            case DataBlockSubtype.ACCELERATION:
+                return "ACCELERATION"
+            case DataBlockSubtype.ANGULAR_VELOCITY:
+                return "ANGULAR VELOCITY"
+            case DataBlockSubtype.GNSS_LOCATION:
+                return "GNSS LOCATION"
+            case DataBlockSubtype.GNSS_METADATA:
+                return "GNSS METADATA"
+            case DataBlockSubtype.HUMIDITY:
+                return "HUMIDITY"
+            case DataBlockSubtype.STATUS:
+                return "STATUS"
+            case _:
+                return "RESERVED"
```

Comment:
I would prefer this be moved back to `data_block.py`; reason being that none of the other enums in this file are required anymore. There is no sensor status, sd card status or deployment state for the current rocket so these should be remove (doesn't necessarily have to be done in this PR but in general).